app/controllers/submissions_controller.rb

```diff
@@ -38,7 +39,8 @@ class SubmissionsController < ApplicationController
   before_filter :authorize_for_student,
                 only: [:file_manager,
                        :populate_file_manager,
-                       :update_files]
+                       :update_files,
+  		       :populate_file_manager_react]
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Tab detected.